frontend_server/BUILD.gn

```diff
@@ -71,4 +74,115 @@ if (is_fuchsia_host || is_fuchsia) {
 
     inputs = frontend_server_files
   }
+
+  # Creates a packageable vm.
+  prebuilt_dart_action("flutter_package_vm") {
```

Comment:
As discussed offline, add comments about these artifacts pointing at the github issue and add a reviewer from the engine team.